src/react/2-intro-to-jsx.md

```diff
@@ -0,0 +1,188 @@
+@page learn-react/intro-to-jsx Introduction to JSX
+@parent learn-react 2
+
+@description Learn how to use React's preferred markup syntax JSX, and combine html with javascript
+
+@body
+
+## Creating Views in React
+
+Before we talk about JSX, let's first discuss exactly how views are created in React.
+
+At it's core, React is a JavaScrpt library used for creating re-usable front-end components. One of the best features of React is that it allows you to describe the way your components look using only JavaScript, there's no HTML or templating languages to keep track of.
+
+But this "all JS" approach comes with some downsides, namely it becomes very difficult to describe complex page hiearacies using only pure JavaScript.
```

Comment:
I'll actually refrain from putting in anymore spelling mistakes because a lot of them can be caught with automated tooling. "hierarchies" instead of "hiearacies". So far so good, though. I'm walking through this, and it looks great @mikedane